packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -437,98 +481,111 @@ function SignInStartInternal(): JSX.Element {
+                  >
+                    <Col gap={6}>
+                      <Form.ControlRow elementId={identifierField.id}>
+                        <DynamicField
+                          actionLabel={nextIdentifier?.action}
+                          onActionClicked={switchToNextIdentifier}
+                          {...identifierFieldProps}
+                          autoFocus={shouldAutofocus}
+                          autoComplete={isWebAuthnAutofillSupported ? 'webauthn' : undefined}
+                        />
+                      </Form.ControlRow>
+                      <InstantPasswordRow field={passwordBasedInstance ? instantPasswordField : undefined} />
+                    </Col>
+                    <Col center>
+                      <CaptchaElement />
+                      <Form.SubmitButton hasArrow />
+                    </Col>
+                  </Form.Root>
+                ) : null}
+              </SocialButtonsReversibleContainerWithDivider>
+              {!standardFormAttributes.length && <CaptchaElement />}
+              {userSettings.attributes.passkey?.enabled &&
+                userSettings.passkeySettings.show_sign_in_button &&
+                isWebSupported && (
+                  <Card.Action elementId={'usePasskey'}>
+                    <Card.ActionLink
+                      localizationKey={localizationKeys('signIn.start.actionLink__use_passkey')}
+                      onClick={() => authenticateWithPasskey({ flow: 'discoverable' })}
+                    />
+                  </Card.Action>
+                )}
+            </Col>
+          </Card.Content>
+          <Card.Footer>
+            {userSettings.signUp.mode === SIGN_UP_MODES.PUBLIC && !isCombinedFlow && (
+              <Card.Action elementId='signIn'>
+                <Card.ActionText localizationKey={localizationKeys('signIn.start.actionText')} />
+                <Card.ActionLink
+                  localizationKey={localizationKeys('signIn.start.actionLink')}
+                  to={clerk.buildUrlWithAuth(signUpUrl)}
+                />
+              </Card.Action>
+            )}
+            {userSettings.signUp.mode === SIGN_UP_MODES.WAITLIST && (
+              <Card.Action elementId='signIn'>
+                <Card.ActionText localizationKey={localizationKeys('signIn.start.actionText__join_waitlist')} />
+                <Card.ActionLink
+                  localizationKey={localizationKeys('signIn.start.actionLink__join_waitlist')}
+                  to={clerk.buildUrlWithAuth(waitlistUrl)}
+                />
+              </Card.Action>
+            )}
+          </Card.Footer>
+        </Card.Root>
+      ) : (
+        <SignInAlternativePhoneCodePhoneNumberCard
+          handleSubmit={handleFirstPartySubmit}
+          phoneNumberFormState={phoneIdentifierField}
+          onUseAnotherMethod={onAlternativePhoneCodeUseAnotherMethod}
+          phoneCodeProvider={alternativePhoneCodeProvider}
+        />
+      )}
```

Comment:
This is the only thing that is actually added on JSX, the rest above is just indentation changes.